### PR TITLE
Toggle chevron direction in TOC

### PR DESCRIPTION
**Pull request by binford2k**
This provides more interaction feedback when opening a menu.

### CODE REVIEW

File: public/js/showoff.js [hunk 1]
```diff
@@ -304,6 +304,7 @@ function setupMenu() {
         .text(slidePath)
         .append(icon)
         .click(function() {
+          $(this).children('i').toggleClass('fa-angle-down fa-angle-up');
```
Comment (marrero984) [inline public/js/showoff.js]: the issue here is that some child elements start expanded and the toggle doesn't validate what state they should be in

Comment (binford2k) [inline public/js/showoff.js]: Do you see a better way than

``` javascript
          if ($(this).next().is(':visible') ) {
            $(this).next().hide();
            $(this).children('i').removeClass('fa-angle-up');
            $(this).children('i').addClass('fa-angle-down');
          }
          else {
            $(this).next().show();
            $(this).children('i').removeClass('fa-angle-down');
            $(this).children('i').addClass('fa-angle-up');
          }
```

Comment (marrero984) [inline public/js/showoff.js]: Not really, which is why I had let it slide for the time being...

**Comment by binford2k** [comment]
This might help, though I don't know exactly where the best place to put it would be. When the menu is built? When it's displayed?

``` javascript
  // style the proper navSection on display.
  $(".navSection").each(function() {
    if ($(this).next().is(':visible') ) {
      $(this).children('i').attr('class', 'fa fa-angle-up');
    }
  });
```

**Comment by marrero984** [comment]
I'll merge with what you have and refactor later.  I've been wanting to take another crack at the menu stuff

**Comment by binford2k** [comment]
@marrero984 totally ready for review & merge now

**Pull request merged**